frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -180,13 +186,31 @@ func toDispatchState(ctx context.Context, dt []byte, opt ConvertOpt) (*dispatchS
 		return nil, err
 	}
 
-	for _, w := range dockerfile.Warnings {
-		opt.Warn(w.Short, w.URL, w.Detail, w.Location)
+	// Moby still uses the `dockerfile.PrintWarnings` method to print warnings,
```

Comment:
follow-up: we can change the Moby codebase as well if needed. We can't just change the implementation so the printing that Moby does becomes impossible to implement.